charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml

```diff
@@ -1,18 +1,9 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    app: kubescape-host-scanner
-    k8s-app: kubescape-host-scanner
-    kubernetes.io/metadata.name: kubescape-host-scanner
-    tier: kubescape-host-scanner-control-plane
-  name: kubescape-host-scanner
 ---
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: host-scanner
-  namespace: kubescape-host-scanner
+  namespace: {{ .Values.ksNamespace }}
```

Comment:
We need forst to upadte kubescape code so it won't delete the host-scanner namespace once it finished.

Reply:
https://github.com/kubescape/kubescape/blob/2a0a2cf95ab59a54ab190179bb33f837a02447bc/core/pkg/hostsensorutils/hostsensordeploy.go#L279

Reply:
And we need to fix the YAML in kubescape itself as well:
https://github.com/kubescape/kubescape/blob/master/core/pkg/hostsensorutils/hostsensor.yaml